indico/modules/categories/controllers/management.py

```diff
@@ -58,8 +60,22 @@ def _category_query_options(self):
         return children_strategy,
 
     def _process(self):
+        show_all_events = request.args.get('show_all') == '1'
```

Comment:
`page=all`? Makes more sense to not have any page number argument when you show all IMO